collector/nodes.go

```diff
@@ -1038,6 +1128,22 @@ func (c *Nodes) Collect(ch chan<- prometheus.Metric) {
 				metric.Labels(nodeStatsResponse.ClusterName, node)...,
 			)
 		}
+		for _, metric := range c.nodeMetricsVec {
+			labelValues := metric.LabelValues(nodeStatsResponse.ClusterName, node)
+			values := metric.Values(node)
+
+			for i, labelValue := range labelValues {
+				counterVec := prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
`NewCounterVec` must not be called in the `Collect` func implementing `prometheus.Collector`. Throughout the collector we are using the pattern to create a new const metric. (`prometheus.MustNewConstMetric`). It perfectly fits to our use case.